src/cgame/cg_rocket_datasource.cpp

```diff
@@ -569,31 +594,16 @@ void CG_Rocket_BuildResolutionList( const char* )
 		Rocket_DSAddRow( "resolutions", "default", buf );
 		rocketInfo.data.resolutionIndex = rocketInfo.data.resolutionCount;
 	}
-
-}
-
-static int ResolutionListCmpByWidth( const void *one, const void *two )
-{
-	resolution_t *a = ( resolution_t * ) one;
-	resolution_t *b = ( resolution_t * ) two;
-
-	if ( a->width > b->width ) return -1;
-
-	if ( b->width > a->width ) return 1;
-
-	if ( a->width == b->width )  return 0;
-
-	return 0; // silence compiler
 }
 
 void CG_Rocket_SortResolutionList( const char*, const char *sortBy )
```

Comment:
CG_Rocket_SortResolutionList could indeed be deleted - it's only useful to have a sort function if the table has more than one column.